pkg/cmd/pulumi/state_move_test.go

```diff
@@ -919,3 +931,101 @@ func TestMoveRootStack(t *testing.T) {
 	assert.Equal(t, urn.URN("urn:pulumi:destStack::test::pulumi:pulumi:Stack::test-destStack"),
 		destSnapshot.Resources[2].Parent)
 }
+
+//nolint:paralleltest // changes directory for process
+func TestMoveSecret(t *testing.T) {
```

Comment:
Should we have a test case for when we're not in the project directory?

Reply:
Done :heavy_check_mark: